frontend/src/admin/words/WordItem.tsx

```diff
@@ -44,15 +43,21 @@ export default function WordItem(props: {
               <span className='md:text-2xl text-purple-400 font-bold'>
                 {props.word.word}
               </span>
-            </div>              
+            </div>
             <div className='word-group md:mt-0 xs:mt-4'>
               <span className='italic'>answer</span>
-              <span
-                className='text-2xl text-purple-400 font-semibold'
-                key={answer?.id}
-              >
-                {answer?.name}
-              </span>
+              {props.word.choices.map((choice: Choice) => {
+                if (choice.is_correct) {
+                  return (
+                    <span
+                      className='md:text-2xl text-purple-400 font-semibold'
+                      key={choice.id}
+                    >
+                      {choice.name}
+                    </span>
+                  );
+                }
+              })}
```

Comment:
Revert back to original code. Usa the `answer` variable

Reply:
na sayop siguro ni ug merge katung pag conflict miss sorry